sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/trees/TreeNode.scala

```diff
@@ -397,6 +393,7 @@ abstract class TreeNode[BaseType <: TreeNode[BaseType]] extends Product {
   def argString: String = productIterator.flatMap {
     case tn: TreeNode[_] if containsChild(tn) => Nil
     case tn: TreeNode[_] => s"${tn.simpleString}" :: Nil
+    case seq: Seq[_] if seq == Seq.empty => "[Empty]" :: Nil
```

Comment:
Thank you! Any suggestion or hint? Or should we revert it back to the original fix? Only overwrite the two Leaf Nodes `LocalTableScan` and `LocalRelation`?